MAUI/CheckBox/Visual-Customization.md

```diff
@@ -261,5 +261,29 @@ The [`ControlSize`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.
     stackLayout.Children.Add(sfCheckBox);
     this.Content = stackLayout;
 
+{% endhighlight %}
+{% endtabs %}
+
+## Font autoscaling enabled
+
+The `FontAutoScalingEnabled` property is used to automatically scale the control's font size based on the operating system's text size. The default value of `FontAutoSaclingEnabled` property is `false`.
```

Comment:
The `FontAutoScalingEnabled` property is used to automatically scale the Radio button's font size based on the operating system's text size. The default value of the `FontAutoSaclingEnabled` property is `false`.

Reply:
Yes, changed.